frontend/src/index.tsx

```diff
@@ -11,7 +11,9 @@ import { ErrorBoundary } from './layout/ErrorBoundary'
 import { loadPostHogJS } from './loadPostHogJS'
 
 loadPostHogJS()
-initKea()
+initKea({
+    prefix: `team_${window.POSTHOG_APP_CONTEXT?.current_team?.id}_project_${window.POSTHOG_APP_CONTEXT?.current_team?.project_id}`,
+})
```

Comment:
we're always putting this info on the window so this will keep all team / project combos separate

Reply:
Doesn't this mean we also won't persist many user preferences cross-project? Like whether the sidebar is collapsed, some banners being acknowledged, replay speed, and such

Reply:
it does... 

i think certainly we should include the team, maybe project is too extreme

Reply:
switched to a simpler thing :)